cypress/support/pages/learning_store/pageLearningStore.js

```diff
@@ -2,12 +2,21 @@
 
 class Learning_Store {
 
-  static #lernStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
+  static #learnStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
+  static #learnstoreMenuLink = '[data-testid="Lern-Store"]'
```

Comment:
Don't you think its redundant? Both selector are referring to the same thing.

``` 
static #learnStoreOverviewNavigationButton = '[data-testid="Lern-Store"]'
static #learnstoreMenuLink = '[data-testid="Lern-Store"]'

```

Reply:
minor typo -> '[data-testid="Lern-Store"]'  -> 'learn_store'